### PR TITLE
Put Ben's instructions into the rep

### PR DESCRIPTION
**Pull request by adrianhopebailie**

### CODE REVIEW

**Comment by emschwartz** [comment]
The commit message should use the standard format, and I think @sharafian should add this so he gets credit for it

**Comment by adrianhopebailie** [comment]
If @sharafian logs a new PR pleaseuse the same file name or change the link I added to the wiki: https://github.com/interledgerjs/ilp-kit/wiki

File: RUN.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+root$ adduser sharafian
+root$ usermod -aG sudo sharafian
+```
+
+Do the rest of this guide while logged in as your new user.
+
+## Postgresql setup
+
+```
+# now do some of the postgresql setup
```
Comment (adrianhopebailie) [inline RUN.md]: Move the comments out of the code block

File: RUN.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+postgres$ createdb ilpkit
+postgres$ exit
+```
+
+## Ilp Kit setup
+
+```
+$ cd
+
+# clone the ilp kit repository and install dependencies.
```
Comment (adrianhopebailie) [inline RUN.md]: Move out of code block

File: RUN.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+- [Enjoy](#enjoy)
+
+# ILP Kit installation
+
+## Get a server instance
+
+### Buy server
+
+- On [DigitalOcean](https://cloud.digitalocean.com/droplets), click `Create Droplet`.
+- Select the option with 2 Gb of RAM (the $20 one).
```
Comment (adrianhopebailie) [inline RUN.md]: Maybe encourage people to use a referral code that earns them $10 credit and you $25 (once they spend $25). Then offer to give that to them as credit on your ledger 😄

Comment (adrianhopebailie) [inline RUN.md]: https://cloud.digitalocean.com/settings/referrals

**Comment by vhpoet** [comment]
See https://github.com/interledgerjs/ilp-kit/pull/76.

**Pull request closed without merging**